front/lib/plans/subscription.ts

```diff
@@ -296,16 +299,45 @@ export const pokeUpgradeWorkspaceToPlan = async (
     );
   }
 
-  if (newPlan.code === PRO_PLAN_SEAT_29_CODE) {
+  const isUgradeToEnterprise = newPlan.code.startsWith("ENT_");
```

Comment:
We have no guarantee that people will follow that nomenclature do we?

Reply:
Should we use `!isPropPlanCode` instead?

Reply:
Poké UI filters the plans from their prefix. 
On the Enteprise modal we display only codes with ENT_
On this flow ENT_ are filtered out.  

There's currently no way to end up here with an ENT plan, I added this to make it clear for a dev that this was not the flow for an Enterprise upgrade. 

I would keep as is for this PR